### PR TITLE
OCPBUGS-4367: Fix missing debug messages when getting baseISO

### PR DESCRIPTION
**Pull request by bfournie**
A new logger is currently created for the baseISO and oc commands. Since the default log level is not debug, this results in the messages not being output. It is cleaner to use the global logger which prints the debug log messages by default.

In addition, a log message was added for a failure running the oc command due to the --icspFile option as this indicates that an older oc binary is being used which doesn't support this option.

### CODE REVIEW

**Comment by zaneb** [comment]
Two more to fix:

* https://github.com/openshift/installer/blob/master/pkg/asset/agent/manifests/nmstateconfig.go#L183
* https://github.com/openshift/installer/blob/master/pkg/asset/agent/manifests/nmstateconfig.go#L271

**Comment by bfournie** [comment]
> Two more to fix:
> 
> * https://github.com/openshift/installer/blob/master/pkg/asset/agent/manifests/nmstateconfig.go#L183
> * https://github.com/openshift/installer/blob/master/pkg/asset/agent/manifests/nmstateconfig.go#L271

Fixed

File: pkg/asset/agent/manifests/nmstateconfig.go [hunk 1]
```diff
@@ -180,9 +180,7 @@ func (n *NMStateConfig) Load(f asset.FileFetcher) (bool, error) {
 		nmStateConfigList = append(nmStateConfigList, &nmStateConfig)
 	}
 
-	log := logrus.New()
-	log.Level = logrus.WarnLevel
```
Comment (zaneb) [inline pkg/asset/agent/manifests/nmstateconfig.go:184]: I'm remembering now the below call is to a library that logs a bunch of stuff at INFO level that we don't want to see: https://github.com/openshift/installer/commit/166434db40d0d399b336f1994d2677f7b59da8f8
We might need to save and restore the level.

Comment (bfournie) [reply pkg/asset/agent/manifests/nmstateconfig.go:184]: Yeah now we get:
```
INFO[0000] Start configuring static network for 5 hosts  pkg=manifests
INFO[0000] Adding NMConnection file <eno1.nmconnection>  pkg=manifests
INFO[0000] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
INFO[0001] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
INFO[0002] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
INFO[0002] Adding NMConnection file <enp1s0.nmconnection>  pkg=manifests
```

Using this doesn't seem to work in hiding those Info messages though, I may need to revert it to use the prior way.
```
        level := logrus.GetLevel()
        logrus.SetLevel(logrus.WarnLevel)
        staticNetworkConfigGenerator := staticnetworkconfig.New(logrus.WithField("pkg", "manifests"), staticnetworkconfig.Config{MaxConcurrentGenerations: 2})
        logrus.SetLevel(level)
```

Comment (zaneb) [reply pkg/asset/agent/manifests/nmstateconfig.go]: Oh, the logging is done by the actual calls to the returned structure, not by `New()`. So we need to do `defer logrus.SetLevel(level)` instead.

Comment (bfournie) [reply pkg/asset/agent/manifests/nmstateconfig.go:184]: Yeah using defer works, thanks. Updated.

**Pull request merged**